docs/table_design/Data_distribution.md

```diff
@@ -59,38 +59,40 @@ For example, if you create a table without specifying the data distribution meth
 
 ```SQL
 CREATE TABLE site_access(
-    site_id INT DEFAULT '10',
-    city_code SMALLINT,
-    user_name VARCHAR(32) DEFAULT '',
-    pv BIGINT SUM DEFAULT '0'
+    event_day DATE,
+    site_id INT DEFAULT '10', 
+    pv BIGINT DEFAULT '0' ,
+    city_code VARCHAR(100),
+    user_name VARCHAR(32) DEFAULT ''
 )
-AGGREGATE KEY(site_id, city_code, user_name); -- The data distribution method is not specified.
+DUPLICATE KEY(event_day,site_id,pv); -- The data distribution method is not specified.
 ```
 
 Specify Hash distribution as the data distribution method at table creation.
 
 ```SQL
 CREATE TABLE site_access(
+    event_day DATE,
     site_id INT DEFAULT '10',
     city_code SMALLINT,
     user_name VARCHAR(32) DEFAULT '',
     pv BIGINT SUM DEFAULT '0'
 )
-AGGREGATE KEY(site_id, city_code, user_name)
-DISTRIBUTED BY HASH(site_id); -- Set the bucketing method as Hash bucketing and specify the bucketing key.
+AGGREGATE KEY(event_day, site_id, city_code, user_name)
+DISTRIBUTED BY HASH(event_day,site_id); -- Set the bucketing method as Hash bucketing and specify the bucketing key.
 ```
 
 Specify Range + Random distribution as the data distribution method at table creation.
 
 ```SQL
 CREATE TABLE site_access(
     event_day DATE,
-    site_id INT DEFAULT '10',
+    site_id INT DEFAULT '10', 
+    pv BIGINT DEFAULT '0' ,
     city_code VARCHAR(100),
-    user_name VARCHAR(32) DEFAULT '',
-    pv BIGINT SUM DEFAULT '0'
+    user_name VARCHAR(32) DEFAULT ''
 )
-AGGREGATE KEY(event_day, site_id, city_code, user_name)
+DUPLICATE KEY(event_day,site_id,pv)
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/ececebdc-9c76-40cc-b364-fb15a5f439f7)
seems the command cannot be run